app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -126,6 +135,7 @@ fun RecentlyVisited(
 private fun RecentlyVisitedHistoryGroup(
     recentVisit: RecentHistoryGroup,
     menuItems: List<RecentVisitMenuItem>,
+    enabled: Boolean,
```

Comment:
I don't have a good name for this, but I think we should rename this to something like `clickableEnabled` to not confuse that there is a disabled ui state.